### PR TITLE
Adding start date and end date attributes to rate card.

### PR DESCRIPTION
**Pull request by sreekanthmandadi**

### CODE REVIEW

**Review by zugdud — CHANGES_REQUESTED** [review]
Hi Sreekanth, please remove the .gitignore file

File: endpoints/schemas/inventorySchemas.yaml [hunk 1]
```diff
@@ -184,6 +184,12 @@ components:
                           type: string
                         rateCardName:
                           type: string
+						rateCardStartDate:
```
Comment (vadimi) [inline endpoints/schemas/inventorySchemas.yaml]: the formatting is off a bit, I'm afraid it might impact the codegen

**Review by sreekanthmandadi — COMMENTED** [review]
Corrected the formatting in the Yaml file

File: endpoints/schemas/inventorySchemas.yaml (hunk 1 above)
Comment (sreekanthmandadi) [reply endpoints/schemas/inventorySchemas.yaml]: Corrected the formatting. Please verify.

**Comment by sreekanthmandadi** [comment]
> Hi Sreekanth, please remove the .gitignore file
gitignore file is removed.

**Pull request merged**